lib/cisco_node_utils/cmd_ref/show_version.yaml

```diff
@@ -65,7 +65,8 @@ system_image:
   N6k: *sys_img_isan
   N7k: *sys_img_isan
   ios_xr:
-    get_value: '/IOS XR.*Version (.*)$/'
```

Comment:
We decided this was a better string to describe "system image" than just the simple version.